docs/DeveloperGuide.md

```diff
@@ -106,45 +118,29 @@ Priority | As a... | I want to... | So that I can...
 high | teacher | add a contact | have a consolidated list of contacts that I require
```

Comment:
are you able to change line 107 to say "schedules" instead of "schedules and tasks". idk but i kinda wanna make it clear that we use "entries" and "schedules" interchangeably. is there a way to specify this somewhere? i'll leave it to you documentation ic